.github/workflows/release-and-publish.yml

```diff
@@ -0,0 +1,45 @@
+name: Draft Release & Publish
+on: [workflow_dispatch]
```

Comment:
can we make this fire on pr merge into main so we won't have to manually fire this workflow each time?

Reply:
Yes, because developers cannot ghost release after anymore, a PR for `changelog` will be first merged and then just go to action and hit release.